src/utils/duration-humanizer.ts

```diff
@@ -0,0 +1,40 @@
+import * as humanizeDuration from 'humanize-duration';
+
+const getTimeInText = (seconds: number, uiConfig: any, round: boolean = false): string | undefined => {
```

Comment:
is this copied from the previous implementation that used humanizeDuration or is something unique in this method compared to the previous implementation ?

Reply:
it's the same as Lian implemented, i just added round argument (boolean) so you can sent it to humanize-duration api and it will round the result to integer number.